scripts/src/public/services/offer_contents.ts

```diff
@@ -1,7 +1,10 @@
 import { LoggerInstance } from "winston";
+import { Request as ExpressRequest } from "express-serve-static-core";
 
 import { isNothing } from "../../utils";
 import * as db from "../../models/offers";
+import { OfferTranslation } from "../../models/translations";
+import { Path } from "../../admin/translations";
```

Comment:
I wouldn't put `Path` in admin if you're importing it into the public server. it should probably be the opposite

Reply:
just define it here `scripts/src/models/translations.ts`